servers/text_server.h

```diff
@@ -79,7 +79,8 @@ class TextServer : public Object {
 		GRAPHEME_IS_BREAK_SOFT = 1 << 5, // Is line break (optional break, e.g. space).
 		GRAPHEME_IS_TAB = 1 << 6, // Is tab or vertical tab.
 		GRAPHEME_IS_ELONGATION = 1 << 7, // Elongation (e.g. kashida), glyph can be duplicated or truncated to fit line to width.
-		GRAPHEME_IS_PUNCTUATION = 1 << 8 // Punctuation (can be used as word break, but not line break or justifiction).
+		GRAPHEME_IS_PUNCTUATION = 1 << 8, // Punctuation, except underscore (can be used as word break, but not line break or justifiction).
+		GRAPHEME_IS_UNDERSCORE = 1 << 9 // Underscore (can be used as word break).
```

Comment:
Nitpick, but I'd add a trailing comma so that future additions don't create additional diff on the previous last entry.